scripts/deploy-supernet.ts

```diff
@@ -615,9 +623,36 @@ async function main() {
   for (let i = 0; i < numberPerMint; i++) {
     await aavegotchiGameFacet.claimAavegotchi(i, 0, ethers.utils.parseEther("10000"));
     const gotchi = await aavegotchiFacet.getAavegotchi(i);
-    console.log(`Aavegotchi ID: ${i} status is claimed (${gotchi.status == 3}) `);
+    console.log(`Aavegotchi ID: ${i} status is claimed (${gotchi.status == 3})`);
+    console.log('aavegotchi owner', gotchi.owner)
   }
 
+  for (let i = 1; i < 200; i++) {
+    try {
+      tx = await ghstTokenContract.mint(ownerAddress, ethers.utils.parseEther('100000000000000000000000'))
+      await tx.wait()
+
+      tx = await ghstTokenContract.approve(shopFacet.address, ethers.utils.parseEther('100000000000000000000000'))
+      await tx.wait()
+
+      tx = await shopFacet.purchaseItemsWithGhst(ownerAddress, [i], [1])
+      await tx.wait()
+      console.log(`purchased item with id ${i}`)
+
+    } catch (e) {
+    }
+  }
+
+  console.log('itemBalancesWithTypes')
+  console.log(await itemsFacet.itemBalancesWithTypes(ownerAddress))
```

Comment:
@narayanprusty
Here is the first situation I mentioned, even though I'm buying a lot of items in the code above the itemBalancesWityTypes function return an empty array